pkgs/applications/graphics/pixinsight/default.nix

```
@@ -0,0 +1,127 @@
+{ stdenv, lib, requireFile, wrapQtAppsHook, autoPatchelfHook, makeWrapper, unixtools, fakeroot
+, mime-types, libGL, libpulseaudio, alsa-lib, nss, gd, gst_all_1, nspr, expat, fontconfig
+, dbus, glib, zlib, openssl, libdrm, cups, avahi-compat, xorg, wayland, libudev0-shim
+# Qt 5 subpackages
+, qt5
```

Since you used `libsForQt5.callPackage`, the `qt5` subpackages should already be in scope:
```suggestion
, qtbase, qtgamepad, qtserialport, qtserialbus, qtvirtualkeyboard, qtmultimedia, qtwebkit
```